roles/manage_operating_system/tasks/enable_core_dump.yml

```diff
@@ -0,0 +1,15 @@
+---
+- name: Enable unlimited core size for all users
+  community.general.pam_limits:
+    domain: "*"
+    limit_type: "-"
+    limit_item: "core"
+    value: "unlimited"
+  become: true
+
+- name: Install debuginfo helper packages
+  ansible.builtin.package:
+    name: yum-utils
+    state: present
+  when: ansible_os_family == 'RedHat'
+  become: true
```

Comment:
the one thing i have with placing this here is about needing to run this `manage_operating_system` role prior to the `install_dbserver` role, i think the install tasks should be run in the same place

Reply:
I think we can move it somewhere else, but it needs a more general location than `install_dbserver` because it can apply to non-database systems.  Do you think an `install_os` role or something is needed? (I don't think install_os is a good name...)

Reply:
I understand, I think leaving it here is a good place.